servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Single.java

```diff
@@ -792,13 +791,41 @@ public final Publisher<T> concat(Publisher<? extends T> next) {
      * replayable {@link Publisher}(s) or when eager subscribe is beneficial.
      * @return New {@link Publisher} that first emits the result of this {@link Single} and then subscribes and emits
      * all elements from {@code next} {@link Publisher}.
+     * @deprecated Use {@link #concat(Publisher)} or {@link #concatDeferSubscribe(Publisher)} instead.
      */
+    @Deprecated
```

Comment:
I'm mixed on this... imho using method overload language feature avoids callers needing to have conditional code if they need to switch between different options. however we have both variants now (qualified name and method overload) so I'm not opposed to this.